lib/core/sdam/server_selection.js

```diff
@@ -298,18 +297,10 @@ function selectServers(topology, selector, timeout, start, callback) {
   }
 
   const retrySelection = () => {
-    // clear all existing monitor timers
-    drainTimerQueue(topology.s.monitorTimers);
-
     // ensure all server monitors attempt monitoring soon
-    topology.s.servers.forEach(server => {
-      const timer = setTimeout(
-        () => server.monitor({ heartbeatFrequencyMS: topology.description.heartbeatFrequencyMS }),
-        TOPOLOGY_DEFAULTS.minHeartbeatFrequencyMS
-      );
-
-      topology.s.monitorTimers.add(timer);
-    });
+    topology.s.servers.forEach(server =>
+      server.monitor({ heartbeatFrequencyMS: topology.description.heartbeatFrequencyMS })
```

Comment:
Are we no longer tracking the monitor timers?

Reply:
Each server has a single `monitorId` attached to it, which drives timed monitoring events. These `monitorTimers` were only for monitor calls made from server selection retries (a timeout to eventually call `server.monitor()`). It turns out we don't need to do this, so we can get rid of a whole queue.